build/config/BUILDCONFIG.gn

```diff
@@ -279,6 +279,15 @@ if (target_os == "android") {
   assert(false, "Unsupported target_os: $target_os")
 }
 
+if (is_cobalt && _default_toolchain == host_toolchain) {
```

Comment:
Let's add a comment explaining why we do this

Reply:
done